.travis.yml

```diff
@@ -12,6 +12,10 @@ env:
     - PRESET=ARM_REL
     - PRESET=ARM_REL_MIN
 
+    - PRESET=CC3200_REL_MIN
+      XI_MAKETARGET=""
+    - PRESET=CC3200_REL_MIN_SECURESOCKET
+      XI_MAKETARGET=""
```

Comment:
I'm not sure min max are good presets for the cc3200.  Plus, couple with secure socket implementation and the matrix is a confusing burden.

How about just two cc3200 presets.

CC3200 (this is what currently is min)
And
CC3200_ONCHIP_TLS

Also I hate to call it secure socket. No realize that it's due to the naming of the simple link API, but here it makes it seem more secure and it's actually not.